From 5340559ad33faf6ff13ea83eb6da3090ff1d5df9 Mon Sep 17 00:00:00 2001
From: JustSoup321 <brandonboese@protonmail.com>
Date: Tue, 11 Jul 2023 17:15:50 -0400
Subject: [PATCH 3/5] Add mtkclient as a flashing option.

---
 pmb/config/__init__.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pmb/config/__init__.py b/pmb/config/__init__.py
index f16894e4..6484fdb3 100644
--- a/pmb/config/__init__.py
+++ b/pmb/config/__init__.py
@@ -884,6 +884,7 @@ flash_methods = [
     "none",
     "rkdeveloptool",
     "uuu",
+    "mtkclient",
 ]
 
 # These folders will be mounted at the same location into the native
-- 
2.39.2

